vms/administrator/services.py

```diff
@@ -0,0 +1,17 @@
+from django.core.exceptions import ObjectDoesNotExist
+from administrator.models import Administrator
+
+
+def get_administrator_by_id(admin_id):
+
+    is_valid = True
+    result = None
+
+    try:
+        administrator = Administrator.objects.get(pk=admin_id)
+    except ObjectDoesNotExist:
```

Comment:
`except Administrator.ObjectDoesNotExist:`

Reply:
@tonythomas01 Same has been done in get_volunteer_by_id in volunteer/services.py . Do I change this then?